docs/arch/adr-24.md

```diff
@@ -0,0 +1,64 @@
+
+- Some shapes have default values which an create bias. For example, the ellipse has a default axis ratio of 0.5 and many volunteers have left the default creating a bias ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tool has peformance impact on the browser as the drawing is being created and with the job to create classification exports as well. This is because the current annotation consists of every single x, y point created
+
+## Decision
+
+The shape's mark annotation models should change for consistency and improved post-classification analysis in the following ways:
+
+- The annotation should use the mathematical standard of _RHC_ with a domain of `(-180, 180]` for consistent angle calculation
+- The annotation model should use `angle` for naming rotation angles. This replaces usage of `rotation`.
+- The annotation model should replace `x` and `y` with `x_center` and `y_center` for shapes where that is applicable
+  - The exceptions are non-shapes like point, line, and transcription line tools, and non-symmetric shapes like fan.
+  - Shape clustering in aggregation is always done with the center
+- All rotations should be defined about `x_center` and `y_center` point. If the rotation cannot be defined around the center point, then the point used should be clearly recorded in the annotation as `x_rotation` and `y_rotation`
+  - Conditional logic in component code can be avoided by using a Mobx-state-tree's computed view functions to get either `x_center` or `x_rotation` mapped to `mark.x`. 
+- Default values should be removed wherever possible. This may result in some tools drawing UX changing. 
+  - The tools that have defaults are ellipse, rotate rectangle, fan. 
+  - A proposed example for the ellipse may involve requiring the volunteers to click twice to set the the two radii. ([comment](https://github.com/zooniverse/front-end-monorepo/issues/500#issuecomment-516788821))
+- The freehand drawing tools mark annotation will be a string of the SVG's path and it will be the responsibility of post-classification analysis to convert this to usable x,y points. We will include a sample script in the `DataDigging` repo for project owners to reference on how to do this. Aggregation in Caesar will have to be updated to do the conversion first.
+- `tool` will change to `toolIndex` to clarify it is referring to the index of the input from the task area. The `toolIndex` is useful to distinguish between multiple instances of the same tool tip in a drawing task.
+- Tasks and drawing tools will have a `type` attribute that map to an enumeration of the type of task or tool like `drawing` or `point`, `ellipse`, etc respectively. This enables the aggregation for Caesar code to auto-configure which extractor to use without checking data types ([comment](https://github.com/zooniverse/front-end-monorepo/issues/823#issuecomment-493896524)).
```

Comment:
Just noting that the models already have `type` defined on them, but I wanted to include that here as having a purpose and this is new to the models compared to PFE.

Reply:
Actually, I confused myself about this. Task models have `type`, but annotations do not, so this will need to be added.

Reply:
Annotation type could be useful if it saves a task lookup to get the type.

I thought marks had a type too, but I don't see it in [the README](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/plugins/drawingTools#mark-models).

This ADR is specifically for the drawing task and drawing annotation but it sounds like it is starting to recommend changes for other task and annotation models?

Reply:
This was my bad and confusion on my part. We recorded the purpose of the `type` for use with the MST stores in past ADRs, but not that we wanted to include it in the annotations. I asked in an RFC and @CKrawczyk said it would be useful, but it didn't end up in the past ADR. I'm happy to write up a separate one on general annotation format changes.

Reply:
Sounds good to me. I could have sworn that marks, at least, have a type already.

Reply:
I did too! But I double checked again just now and tasks and tools do have types, but annotations and marks do not.

Reply:
I did another pass on this statement to clarify even more what the proposed change is.